.github/workflows/pythonpackage.yaml

```diff
@@ -0,0 +1,27 @@
+name: Tests of renewablepotentialslib
+
+on: [push, pull_request]
+
+jobs:
+    test:
+        name: Run tests
+        runs-on: ubuntu-latest
+        defaults:
+            run:
+                shell: bash -l {0}
+        steps:
+            - uses: actions/checkout@v2
+            - name: Setup conda environment
+              uses: conda-incubator/setup-miniconda@v2
+              with:
+                auto-update-conda: true
+                python-version: 3.6
+                add-pip-as-python-dependency: true
+            - name: Install renewablepotentialslib and test requirements.
```

Comment:
You can merge this step with the one above. See my changes in https://github.com/calliope-project/euro-calliope/pull/70/files.